app/index.js

```diff
@@ -84,6 +83,10 @@ module.exports = class extends Generator {
     this.fs.writeJSON(this.destinationPath('package.json'), pkg);
   }
 
+  conflicts() {
+    this.fs.append(this.destinationPath('.eslintignore'), '**/templates\n');
```

Comment:
I think append automatically handle whitespace, but I might be wrong. Can you double check?